src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -245,6 +270,24 @@ export default {
 		if (expCookieSignifier === 'b') {
 			this.$kvTrackEvent('Borrower Profile', 'EXP-GROW-655-Aug2021', expCookieSignifier);
 		}
+		// EXP-MARS-95-May2022
+		this.apollo.watchQuery({ query: publicSocialShareQuery }).subscribe({
+			next: ({ data }) => {
+				const version = _get(data, 'experiment.version');
```

Comment:
Can we convert this to use optional chaining and remove the `_get` import?

Reply:
Right away